File: programmers/난이도별/level01.완주하지_못한_선수/JiaLee0707.java
import java.util.*;

class Solution {
    public String solution(String[] participant, String[] completion) {
        Arrays.sort(participant);
        Arrays.sort(completion);
        String answer = participant[participant.length-1];
        
        for(int i=0; i<completion.length; i++) {
            if(!participant[i].equals(completion[i])) {
                answer = participant[i];
                break;
            }
        }
        return answer;
    }
}

File: programmers/난이도별/level01.두_개_뽑아서_더하기/JiaLee0707.js
function solution(numbers) {
    let answer = [];
    const numbers_length = numbers.length;
    
    for(let i=0; i<numbers_length; i++) {
        for(let j=i+1; j<numbers_length; j++) {
            answer.push(numbers[i] + numbers[j]);
        }
    }
    
    answer = [...new Set(answer)];
    answer.sort((a, b) => a - b);
    
    return answer;
}
